samples/hybrid-connections/node/hyco-websocket-tunnel/server.js

```diff
@@ -57,11 +57,11 @@ wsServer.on('request', function(request) {
   var args = url.pathname.split('/').slice(1);
```

Comment:
>url [](start = 13, length = 3)

Where does this url get built? Why doesn't it seem to have /tunnel/ in the path?